parm/soca/obs/config/icec_viirs_n20_l2_south.yaml

```diff
@@ -0,0 +1,45 @@
+obs space:
+  name: icec_viirs_n20_l2_south
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: !ENV ${DATA}/obs/${OPREFIX}icec_viirs_n20_l2_south.{PDY}${cyc}.nc4
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: !ENV ${DATA}/diags/icec_viirs_n20_l2_south.${PDY}${cyc}.nc4
+  simulated variables: [seaIceFraction]
+  io pool:
+    max pool size: 1
+get values:
+  time interpolation: linear
+obs operator:
+  name: Identity
+  observation alias file: obsop_name_map.yaml
+obs error:
+  covariance model: diagonal
+obs filters:
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_area_fraction}
+    minvalue: 0.9
+- filter: Bounds Check
+  minvalue: 0.0
+  maxvalue: 1.0
+#- filter: Background Check
+#  threshold: 5.0
+- filter: Domain Check
```

Comment:
Why is there a domain check for sea_surface_temperature > 2.0 and also > 0? Isn't one sufficient?